### DISCUSSION TITLE
Help setting up Next.js with Prisma and PostgreSQL

### DISCUSSION THREAD

**Discussion by SinghAstra** [vercel/next.js#70720] [Help] [answered]
### Summary

I come from a **MERN background**, and I'm struggling to set up the `DATABASE_URL` for PostgreSQL. I know it should look something like this:

```env
DATABASE_URL="postgresql://username:password@localhost:5432/dbName"
```

However, I’m confused about a few things:

1. Do I need to create my own username and password, or is there a standard way to set this up?
2. Is there a GUI for viewing the data uploaded to the PostgreSQL database?
3. What’s the equivalent of `console.log("Connected to the MongoDB database successfully")` in this setup?

Any help or resources would be greatly appreciated!

**Comment by wilsonadenuga** [comment, marked as answer]
> ### Summary
> I come from a **MERN background**, and I'm struggling to set up the `DATABASE_URL` for PostgreSQL. I know it should look something like this:
> 
> ```dotenv
> DATABASE_URL="postgresql://username:password@localhost:5432/dbName"
> ```
> 
> However, I’m confused about a few things:
> 
> 1. Do I need to create my own username and password, or is there a standard way to set this up?
> 2. Is there a GUI for viewing the data uploaded to the PostgreSQL database?
> 3. What’s the equivalent of `console.log("Connected to the MongoDB database successfully")` in this setup?
> 
> Any help or resources would be greatly appreciated!
> 
> ### Additional information
> _No response_
> 
> ### Example
> _No response_

If you want to run a local database, you can download and use [pgAdmin](https://www.pgadmin.org/), 
you can also get started with a free serverless postgres db from [https://neon.tech](https://neon.tech) works similar to MongoDB cloud

**Comment by suprunchuk** [comment]
There are several GUI tools available for interacting with PostgreSQL databases. Some popular ones include:

pgAdmin: A web-based interface for managing PostgreSQL databases.
DBeaver: A universal database tool that supports PostgreSQL and many other databases.
DataGrip: A database IDE from JetBrains that supports PostgreSQL.
